microphone_array/mic_array_adapter_hw.tcl

```diff
@@ -0,0 +1,128 @@
+# TCL File Generated by Component Editor 18.0
+# Wed Aug 12 08:11:41 MDT 2020
+# DO NOT MODIFY
+
+
+# 
+# mic_array_adapter "mic_array_adapter" v1.0
+#  2020.08.12.08:11:41
+# 
+# 
+
+# 
+# request TCL package from ACDS 16.1
+# 
+package require -exact qsys 16.1
+
+
+# 
+# module mic_array_adapter
+# 
+set_module_property DESCRIPTION ""
+set_module_property NAME mic_array_adapter
+set_module_property VERSION 1.0
+set_module_property INTERNAL false
+set_module_property OPAQUE_ADDRESS_MAP true
+set_module_property AUTHOR ""
+set_module_property DISPLAY_NAME mic_array_adapter
+set_module_property INSTANTIATE_IN_SYSTEM_MODULE true
+set_module_property EDITABLE true
+set_module_property REPORT_TO_TALKBACK false
+set_module_property ALLOW_GREYBOX_GENERATION false
+set_module_property REPORT_HIERARCHY false
+
```

Comment:
Looking good, we just need to add the "set_module_property GROUP "FPGA Open Speech Tools"" tag to this file